### PR TITLE
Add extraction of high-quality docs to text files

### LINKED ISSUE #5: Extraction of texts quality=HIGH and given category
The example shows the extraction of text quality=HIGH and a given category. The result is saved to disk as a folder with txt files

### PR DESCRIPTION
**Pull request by MarekGarbowski**
This commit introduces a new function save_high_quality_docs() in the extraction_to_txt_file.py. It saves high-quality documents from the Speakleash dataset to a specified folder, creating the necessary directories if they don't exist. The .gitignore file has been updated to exclude the output directory from the repository. This function helps in segregating high quality documents for further analysis.

### CODE REVIEW

File: example_3_quality_metrics/extraction_to_txt_file.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import os
```
Comment (IgorTest19) [inline example_3_quality_metrics/extraction_to_txt_file.py]: Missing module docstring.

File: example_3_quality_metrics/extraction_to_txt_file.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+from speakleash import Speakleash
+
+
+def save_high_quality_docs(txt_folder_name):
```
Comment (IgorTest19) [inline example_3_quality_metrics/extraction_to_txt_file.py]: You can split up functionalities using 2 functions:
- `def get_datatxt_folder_name)` which creates directory
- `def save_quality_docs(quality='HIGH')` which will wrap code from creating an instance of the Speakleash class (from line 26 to 41)
- use these two functions inside of condition:` if __name__ == '__main__':`

Look at the following example:
https://github.com/speakleash/speakleash-examples/pull/13/files (file charts.py)

File: .gitignore [hunk 3]
```diff
@@ -157,6 +157,7 @@ cython_debug/
 #  be found at https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore
```
Comment (IgorTest19) [inline .gitignore]: Remove .gitignore file. It's not needed there :)

**Review by IgorTest19 — CHANGES_REQUESTED** [review]

**Comment by MarekGarbowski** [comment]
The extraction_to_txt_file script has been revised for added versatility. A new module has been added providing functions to create necessary directories and save documents from the 'speakleash' dataset to a specific folder. The 'save_high_quality_docs' method was updated to 'save_quality_docs' to allow for selection of document quality. New usage information was also added for guidance on the script modification, importing and using the module's functions, running the script directly, and the output from the script execution. A 'get_data' function was also added to manage the creation of necessary directories. This update provides improved script functionality and versatile document retrieval.

**Review by IgorTest19 — APPROVED** [review]

**Comment by MarekGarbowski** [comment]
Added README.md file and both moved to separate folder.

**Comment by MarekGarbowski** [comment]
Modified the readme.md and main files as suggested.

**Review by IgorTest19 — APPROVED** [review]

**Pull request merged**